content/pages/release-signing.md

```diff
@@ -695,7 +695,10 @@ Projects may make use of automated signing for artifacts built by a CI system su
 
 - All artifacts being signed can be built <a href="https://reproducible-builds.org" target="_blank">reproducibly</a>
 - CI deploys the artifacts to a staging environment
-- The release procedure contains a validation step where all artifacts are reproduced on <a href="https://www.apache.org/legal/release-policy.html#owned-controlled-hardware" target="_blank">trusted hardware</a> before publication to pages intended for end users
+- The release procedure contains a validation step where all artifacts are reproduced on <a href="https://www.apache.org/legal/release-policy.html#owned-controlled-hardware" target="_blank">trusted hardware</a> (i.e. not GitHub Actions) before publication to pages intended for end users. This means rebuilding the artifacts from source and validating the resulting artifacts are bit-by-bit identical to those staged for release.
```

Comment:
```suggestion
- The release process contains a validation step where all artifacts are reproduced and validated on <a href="https://www.apache.org/legal/release-policy.html#owned-controlled-hardware" target="_blank">trusted hardware</a> (i.e. not GitHub Actions) before publication to pages intended for end users. This means rebuilding the artifacts from source and validating the resulting artifacts are bit-by-bit identical to those staged for release.
```

Just to clarify that it's about the release process and not a "code procedure" like a GH action.